README.md

```diff
@@ -241,3 +241,49 @@ or
 **Note**: If running on Docker and/or Docker Machine, substitute the hostnames
 and ports of the above URLs with your Docker settings, as necessary (see the
 [running with Docker section](#how-to-run-with-docker) for more help).
+
+
+### Register a Right allowed by another Right or Copyright
+
+Rather than registering new Rights directly to resulting rightsholders, new
+Rights are initially registered to the current holder of the existing Right or
+Copyright that allows for the new Right's creation. Doing so allows the initial
+link between the source rightsholder and resulting rightsholder to be kept as
+part of the new Right's chain of provenance.
+
+Note that the attributes for the `right` may be much more diverse; see the COALA
+IP models definition (not yet publicly available yet).
+
+Also see [transfering a Right](FIXME) on how to transfer a registered Right to
+new holders.
+
+```
+POST /api/v1/right/
```

Comment:
URLs to endpoints should always be in plural form: `rights`.

I just noticed that we've made that mistake with manifestation as well :/